frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/_add-capacity-modal.scss

```diff
@@ -1,36 +1,46 @@
 @import '~@patternfly/patternfly/sass-utilities/colors';
 
+.add-capacity-modal__hide {
```

Comment:
There are more CSS changes in this PR than #3252.  Is that intentional?  Seems like CSS changes from other PRs is sneaking in here.

Reply:
ack